src/caret_analyze/infra/lttng/id_remapper.py

```diff
@@ -101,3 +101,108 @@ def get_object_id(
             return max_item.remapped_id
         else:
             return addr
+
+
+class AddressRemapper:
```

Comment:
I suggest changing name of this class to `IDRemapperCollection` to better represent it's role.
Variable name `_remapper` looks good to me even after this change.

Reply:
Fixed in [1d9ae6e](https://github.com/tier4/caret_analyze/pull/455/commits/1d9ae6e9cab36add6e776e71f2a5443db1d64d4f).